### ISSUE TITLE
fix this module

### ISSUE THREAD

**Issue by daviddias** [diasdavid/github-to-omnifocus#1]
This module uses [osa-omnifocus](https://github.com/wtfaremyinitials/osa-omnifocus/) which stopped working with omnifocus 3. osa-omnifocus uses https://www.npmjs.com/package/osa.

Found https://discourse.omnigroup.com/t/omnifocus-3-applescript-guide/41915 which describes how to use Omnifocus 3 with AppleScript.

**Comment by daviddias** [comment]
tracking: https://discourse.omnigroup.com/t/github-to-omnifocus/44233

**Comment by daviddias** [comment]
[WIP](https://github.com/daviddias/github-to-omnifocus/tree/fix)

**Comment by daviddias** [comment]
Every once in a while I like to come back to this module with the hopes of automating this part of my life again. Today was one of those times.

I did some upgrades on the https://github.com/daviddias/github-to-omnifocus/tree/fix branch. The code now is more clean, it uses async await for pulling things from github and deps have been updated. 

I also spent some time attempting to solve the interaction with Omnifocus  (the big crux of this issue) and tried both the `osa2` and `jxa` module to make it work. I learned that there is not a single module on NPM that interacts with Omnifocus 3 successfully, they all fail. 

<img width="1114" alt="image" src="https://user-images.githubusercontent.com/1211152/64324550-4091fd00-cfcf-11e9-833e-6004ac13ddc0.png">

Of course, some of them fail due to their age (only compatible with Omnifocus 2, for example). But even after spelunking, updating deps and trying to fix the code, using OSA to interact with Omnifocus 3 continues to be a blackbox of many trials and errors.
